### ISSUE TITLE
GLB/GLFT model with animations shows only bones until the scene is reloaded

### ISSUE THREAD

**Issue by joaopedrosgs** [godotengine/godot#55985]
### Godot version

4.0 dev (e31138958)

### System information

Windows 10 19044.1415, Nvidia GTX 1660

### Issue description

Whenever I try to add a glb or gltf model with animations to a scene, the editor throws a bunch of errors and I can only see the bones of the model

What I see right after dragging a glb character model to a scene:
![error](https://user-images.githubusercontent.com/16986475/146347802-2b5a7763-bb8e-470c-af6d-83bb16dbc9e8.png)

What I see when I try the same steps on Godot 3.x:
![image](https://user-images.githubusercontent.com/16986475/146349648-01befaf5-137e-4d61-b4db-5bfe20cd561a.png)

Errors:
```
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::ArrayMesh_tjj8m
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Skin_bj5xb
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_c7ugn
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_m1iwl
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_tusko
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_ieid6
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_l1e2g
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_a3h1i
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_maf2j
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_umfcy
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_smi7w
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_bplgr
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_sgx16
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_8fib1
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_kip1m
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_yghvk
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_2i7vp
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_12l0m
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_vgu0n
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_iv422
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_0yfa3
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_jwr2o
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_085l0
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_4gm4q
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_lr0dn
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_uq05m
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_1ip2n
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_14y2j
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_7ip0p
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_msg6k
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_kii7y
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_6i4m4
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_hywv2
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_m05p6
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_frpmv
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_wj0gr
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_dq3ao
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_0wrw7
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_0iyxx
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_d16fx
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_4giuc
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_74fy8
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_3m6nr
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_bug6a
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_dwtn8
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_jbrj8
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_7k4qe
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_oft8i
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_3rdem
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_w72i6
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_ysxme
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_kpg8b
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_jercy
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_tqpw8
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_1l240
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_femqt
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_ld36r
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_orv0j
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_v0avm
  core/io/resource_format_binary.cpp:326 - Couldn't load resource (no cache): res://player_with_anims.glb::Animation_maw26
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
  scene/animation/animation_player.cpp:1161 - Condition "p_animation.is_null()" is true. Returning: ERR_INVALID_PARAMETER
Create Node

```

### Steps to reproduce

1. Create a project
2. Create a new Node3D as base
2. Add a glb or gltf file to the project folder
3. Wait for the file to be imported
4. Drag the glb or gltf from the FileSystem to the Node3D in the viewport
5. See a bunch of errors

### Minimal reproduction project

[error_glb.zip](https://github.com/godotengine/godot/files/7726186/error_glb.zip)

**Comment by Calinou** [comment]
This is a known issue with newly imported meshes/textures. Saving the scene then using **Scene > Reload Saved Scene** should let you see the imported mesh.

**Comment by joaopedrosgs** [comment]
Seems to have been fixed by #58007
